wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController.swift

```diff
@@ -97,7 +96,7 @@ final class ConversationListViewController: UIViewController {
         self.selfProfileViewControllerBuilder = selfProfileViewControllerBuilder
 
         let bottomInset = ConversationListViewController.contentControllerBottomInset
-        listContentController = ConversationListContentController(userSession: viewModel.userSession)
+        listContentController = .init(userSession: viewModel.userSession)
```

Comment:
For me the use of .init makes the code less readable and harder to understand at a glance, especially when identifying which controller or object is being initiated. Using the full name of the object would improve clarity and comprehension.

Reply:
I'm not sure I agree on this, but let's have a separate discussion eventually, please.